### PR TITLE
0.62 notifications

### LINKED ISSUE #36: Fastq file passed to bcbio are not the merged ones
But they're the input of the merge
I think bcbio_prepare_samples should return the merged files it generates so they can be passed to bcbio
https://github.com/EdinburghGenomics/Analysis-Driver/blob/master/analysis_driver/driver.py#L149

### LINKED ISSUE #38: Quiet mode
We should add a `--quiet` flag that turns off the stdout logger, so we can run the pipeline through cron without redirecting to a file or `/dev/null`

### PR DESCRIPTION
**Pull request by mwhamgenomics**
Closes #38. #19: Simplified use of notification centre. Fixes #36.

### CODE REVIEW

**Comment by tcezard** [comment]
I would have preferred the fix for the bug we identified yesterday to be tagged independently before we add another set of change.
Unless you're convince we're not going to introduce more issue with this merge, could you tag 0.61.1 with the master adding the last bugfix you did in https://github.com/EdinburghGenomics/Analysis-Driver/commit/a6bcc3b43e3f07bd66e59dc52343f03250e3ebc5

**Comment by mwhamgenomics** [comment]
Pushed https://github.com/EdinburghGenomics/Analysis-Driver/commit/6d6bd19e8fc37f865b6610575e60f98a4ec1bcb9 with the glob.glob fix, and deployed as tag 0.61.1

**Comment by mwhamgenomics** [comment]
Should we take out the `#PBS -M` email notifications from the PBSWriter at this point?

**Comment by tcezard** [comment]
Yes I think that a good idea

**Comment by mwhamgenomics** [comment]
That should be ready for merging now.
M.

File: analysis_driver/notification/email_notification.py [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
-    def fail_step(self, step_name, **kwargs):
-        subject = "{} has failed"
-        body = "{} has failed"
+        msg = MIMEText(body, 'plain')
+        msg['Subject'] = subject
+        # msg['From'] = self.reporter_email
+        msg['To'] = ','.join(self.recipient_emails)
+
+        p = subprocess.Popen(
+            ['mail', '-t'],
```
Comment (tcezard) [inline analysis_driver/notification/email_notification.py]: I don't understand the need to go through a subprocess to send the email.
We should use a university mail relay

**Pull request merged**